### PR TITLE
added translations for Babel, JSX, and Build Steps

### PR DESCRIPTION
**Pull request by shobhitsingh29**
- [x] Babel, JSX, and Build Steps

**Before** :
Screenshot 2019-10-02 at 4 11 43 AM
**After** :
Screenshot 2019-10-02 at 4 13 25 AM

### CODE REVIEW

**Comment by shobhitsingh29** [comment]
@arshadkazmi42 can you review this 😄

File: content/docs/faq-build.md [hunk 1]
```diff
@@ -6,15 +6,15 @@ layout: docs
 category: FAQ
 ---
 
-### Do I need to use JSX with React? {#do-i-need-to-use-jsx-with-react}
+### क्या मुझे React के साथ JSX का उपयोग करने की आवश्यकता है? {#do-i-need-to-use-jsx-with-react}
 
-No! Check out ["React Without JSX"](/docs/react-without-jsx.html) to learn more.
+नहीं! अधिक जानने के लिए ["JSX के बिना React करें"](/docs/react-without-jsx.html) देखें ।
```
Comment (arshadkazmi42) [inline content/docs/faq-build.md]: `JSX के बिना React करें` => `JSX के बिना React`

File: content/docs/faq-build.md [hunk 2]
```diff
@@ -6,15 +6,15 @@ layout: docs
...
 
-### Do I need to use ES6 (+) with React? {#do-i-need-to-use-es6--with-react}
+### क्या मुझे React के साथ ES6 (+) का उपयोग करने की आवश्यकता है? {#do-i-need-to-use-es6--with-react}
 
-No! Check out ["React Without ES6"](/docs/react-without-es6.html) to learn more.
+नहीं! अधिक जानने के लिए ["ES6 के बिना React करें"](/docs/react-without-es6.html) देखें ।
```
Comment (arshadkazmi42) [inline content/docs/faq-build.md]: `ES6 के बिना React करें` => `ES6 के बिना React`

How about this?

Comment (shobhitsingh29) [reply content/docs/faq-build.md]: done

File: content/docs/faq-build.md (hunk 1 above)
Comment (shobhitsingh29) [reply content/docs/faq-build.md]: done

**Comment by shobhitsingh29** [comment]
@arshadkazmi42 done with review comments

**Pull request merged**